src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java

```diff
@@ -1503,5 +1491,12 @@ public void run() {
                     20,
                     16);
         }
+
+        if (Chisel.isModLoaded()) {
+            GT_Values.RA.stdBuilder().itemInputs(GT_ModHandler.getModItem(Chisel.ID, "hempcretesand", 1L))
```

Comment:
would I be correct in assuming there is a chisel PR following or do you want both recipes?

Reply:
there will be one following in a min

Reply:
https://github.com/GTNewHorizons/Chisel/pull/31